app/views/shared/_attributes.html.erb

```diff
@@ -1,8 +1,11 @@
-      <table class="table">
+<% include_title ||= false # title is part of the exterior template in common cases %>
```

Comment:
Do we need this conditional? This basically leaves us with a block of uncovered code.